pkg/cloud/azure/actuators/machine/actuator.go

```diff
@@ -128,72 +146,78 @@ func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machi
 	if err != nil {
 		return errors.Errorf("failed to create scope: %+v", err)
 	}
-
 	defer scope.Close()
 
-	computeSvc := compute.NewService(scope.Scope)
-	networkSvc := network.NewService(scope.Scope)
-
 	bootstrapToken, err := a.checkControlPlaneMachines(scope, cluster, machine)
 	if err != nil {
 		return errors.Wrapf(err, "failed to check control plane machines in cluster %s", cluster.Name)
 	}
 
-	kubeConfig, err := a.GetKubeConfig(cluster, nil)
-	if err != nil {
-		return errors.Wrapf(err, "failed to retrieve kubeconfig while creating machine %q", machine.Name)
+	// kubeConfig, err := a.GetKubeConfig(cluster, nil)
+	// if err != nil {
+	// 	return errors.Wrapf(err, "failed to retrieve kubeconfig while creating machine %q", machine.Name)
+	// }
+
+	networkInterfaceSpec := &networkinterfaces.Spec{
+		Name:     fmt.Sprintf("%s-nic", machine.Name),
+		VNETName: azure.DefaultVNETName,
+	}
+	switch set := scope.Machine.ObjectMeta.Labels["set"]; set {
+	case "node":
+		networkInterfaceSpec.SubnetName = azure.DefaultAgentSubnetName
+	case "controlplane":
+		networkInterfaceSpec.SubnetName = azure.DefaultMasterSubnetName
+		networkInterfaceSpec.PublicLoadBalancerName = azure.DefaultPublicLBName
+		networkInterfaceSpec.InternalLoadBalancerName = azure.DefaultInternalLBName
+		networkInterfaceSpec.NatRule = 0
```

Comment:
Prefer capitalizing an acronym except in instances where it's adjacent to another one.